### PR TITLE
PullRequest1

### PR DESCRIPTION
**Pull request by WARL0C**
testing

### CODE REVIEW

File: TestApp1/NewClass2.cs [hunk 1]
```diff
@@ -9,5 +9,7 @@ namespace TestApp1
     public class NewClass2
     {
         public string Prop2 { get; set; }
+
+        public string Prop3 { get; set; }
```
Comment (AridTag) [inline TestApp1/NewClass2.cs]: I really hate the name of this property

Comment (WARL0C) [inline TestApp1/NewClass2.cs]: blah blah

Comment (WARL0C) [reply TestApp1/NewClass2.cs]: will change...

Comment (WARL0C) [reply TestApp1/NewClass2.cs]: I changed... the property name.  please recheck and approve

**Pull request merged**